src/instance/minecraft/chat-manager.ts

```diff
@@ -26,6 +26,8 @@ import UnmuteChat from './chat/unmute.js'
 import type { MinecraftChatMessage } from './common/chat-interface.js'
 import type MinecraftInstance from './minecraft-instance.js'
 
+import { escapeDiscord } from 'src/util/shared-util.js'
+
```

Comment:
Events sent without discord escape. discord escaping only happens on discord side. Events can be received and processed in other places. Even if not right now, In future updates maybe